.gitignore

```diff
@@ -2,3 +2,6 @@
 db/*.sqlite3
 db/jwt-rsa.key
 vendor/
+.idea/
+*.iml
+.vscode
```

Comment:
Are these relevant to Docker?

Reply:
These are just IDE files to be ignored. If you mind I can remove them, but I guess they do not cause any problem.